source/includes/fundamentals/code-snippets/srv.java

```diff
@@ -1,31 +1,40 @@
 package fundamentals;
 
+import com.mongodb.*;
 import org.bson.BsonDocument;
 import org.bson.BsonInt64;
 import org.bson.Document;
 import org.bson.conversions.Bson;
 
-import com.mongodb.MongoClientSettings;
-import com.mongodb.MongoException;
 import com.mongodb.client.MongoClient;
 import com.mongodb.client.MongoClients;
 import com.mongodb.client.MongoDatabase;
 
 public class MongoClientConnectionExample {
     public static void main(String[] args) {
-        // Replace the uri string with your MongoDB deployment's connection string
-        String uri = "mongodb://user:pass@sample.host:27017/?maxPoolSize=20&w=majority";
+        // Replace the placeholders with your credentials
+        String uri = "mongodb+srv://<username>:<password>@cluster0.sample.mongodb.net/?retryWrites=true&w=majority";
 
-        try (MongoClient mongoClient = MongoClients.create(uri)) {
+        // Construct a ServerApi instance using the ServerApi.builder() method
+        ServerApi serverApi = ServerApi.builder()
+                .version(ServerApiVersion.V1)
+                .build();
 
-            MongoDatabase database = mongoClient.getDatabase("admin");
+        MongoClientSettings settings = MongoClientSettings.builder()
+                .applyConnectionString(new ConnectionString(uri))
+                .serverApi(serverApi)
+                .build();
 
+        // Create a new client and connect to the server
+        try (MongoClient mongoClient = MongoClients.create(settings)) {
+            MongoDatabase database = mongoClient.getDatabase("admin");
             try {
+                // Send a ping to confirm a successful connection
                 Bson command = new BsonDocument("ping", new BsonInt64(1));
                 Document commandResult = database.runCommand(command);
-                System.out.println("Connected successfully to server.");
+                System.out.println("Pinged your deployment. You successfully connected to MongoDB!");
             } catch (MongoException me) {
-                System.err.println("An error occurred while attempting to run a command: " + me);
+                System.err.println("An error occurred while attempting to ping: " + me);
```

Comment:
Suggestion:
I think this could just print the exception similar to the [equivalent C#/.NET snippet](https://www.mongodb.com/docs/drivers/csharp/current/fundamentals/connection/connect/#connect-to-atlas):

```
System.err.println(me);
```